pkg/channelserver/handler.go

```diff
@@ -6,16 +6,10 @@ import (
 	rurl "net/url"
 )
 
-var k3sURL = "http://localhost:8115"
-var rke2URL = "http://localhost:8116"
+var rawURL = "http://localhost:8115"
```

Comment:
If this is not used anywhere else besides the function below, can you move it there and make it a const?